packages/@angular/cli/commands/help.ts

```diff
@@ -56,7 +56,7 @@ const HelpCommand = Command.extend({
 
       if (rawArgs.length > 0) {
         if (cmd === rawArgs[0]) {
-          this.ui.writeLine(command.printDetailedHelp(commandOptions));
+          this.ui.writeLine(command.printBasicHelp(commandOptions));
```

Comment:
There is no implementation of `printDetailedHelp` It should be implemented for each task. 
https://github.com/angular/angular-cli/blob/master/packages/%40angular/cli/ember-cli/lib/models/command.js#L433

We haven't implemented it on any except below mentioned. 
- https://github.com/angular/angular-cli/blob/master/packages/%40angular/cli/commands/generate.ts#L41
- https://github.com/angular/angular-cli/blob/master/packages/%40angular/cli/ember-cli/lib/commands/generate.js#L67
- https://github.com/angular/angular-cli/blob/master/packages/%40angular/cli/ember-cli/lib/models/blueprint.js#L1122